website/pages/en/subgraphs.mdx

```diff
@@ -34,8 +40,47 @@ To learn more about each subgraph component, check out [creating a subgraph](/de
 4. [Publish a subgraph](/publishing/publishing-a-subgraph/)
 5. [Signal on a subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph)
 
-## Subgraph Lifecycle
+### Build locally
 
-Here is a general overview of a subgraph’s lifecycle:
+Great subgraphs start with a local development environment and unit tests. Developers use [Graph CLI](https://github.com/graphprotocol/graph-tooling/tree/main/packages/cli), a command-line interface tool for building and deploying subgraphs on The Graph. They can also use [Graph TypeScript](/developing/graph-ts/) and [Matchstick](/developing/unit-testing-framework/) to create robust subgraphs.
 
-![Subgraph Lifecycle](/img/subgraph-lifecycle.png)
+### Deploy to Subgraph Studio
+
+Once defined, a subgraph can be [deployed to Subgraph Studio](/deploying/deploying-a-subgraph-to-studio/). In Subgraph Studio, you can do the following:
+
+- Use its staging environment to index the deployed subgraph and make it available for review.
+- Verify that your subgraph doesn't have any indexing errors and works as expected.
+
+### Publish to the Network
+
+When you're happy with your subgraph, you can [publish it](/publishing/publishing-a-subgraph/) to The Graph Network.
+
+- This is an on-chain action, which registers the subgraph and makes it discoverable by Indexers.
```

Comment:
Do we use "on-chain" or "onchain"? cc @bermchain

Reply:
@benface I'll ask and edit

Reply:
on-chain more common  in web3 and typically used in our docs